modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/node_modules/frontend-js-react-web$classnames@2.2.6/index.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
You probably explained this somewhere in the commit messages or elsewhere, but GitHub is labeling these files as "generated" (probably because of the "node_modules" in the path) and so it hides them.

In Slack you said, "I still need to autogenerate the bridges" — is this what you were referring to? I gather once you do that these won't be committed to the repo, right?

Reply:
Yes. These files are gitignored, that's why github hides them, probably. I need to send another PR to npm-scripts to autogenerate them. Otherwise it is very possible that this doesn't pass the filters. And also, we don't want to scatter garbage all over the portal.

Reply:
> And also, we don't want to scatter garbage all over the portal.

Hm... Maybe we should tell the others, because it would be a significant departure from our historical customs... 🤔

![smug](https://user-images.githubusercontent.com/7074/101794309-0931ca80-3b07-11eb-964d-33913a1504bf.gif)

Reply:
:fire: